src/Sentry.Bindings.Cocoa/Sentry.Bindings.Cocoa.csproj

```diff
@@ -46,44 +48,49 @@
   </ItemGroup>
 
   <!-- Downloads and sets up the Cocoa SDK: dotnet msbuild /t:setupCocoaSDK src/Sentry.Bindings.Cocoa/Sentry.Bindings.Cocoa.csproj -->
-  <Target Name="SetupCocoaSDK"
-          Condition="$([MSBuild]::IsOSPlatform('OSX')) And !Exists('$(SentryCocoaCache)Sentry-Dynamic.xcframework')"
-          BeforeTargets="BeforeBuild">
-
-    <PropertyGroup>
-      <PropertiesContent>$([System.IO.File]::ReadAllText("../../modules/sentry-cocoa.properties"))</PropertiesContent>
-      <CocoaVersion>$([System.Text.RegularExpressions.Regex]::Match($(PropertiesContent), 'version\s*=\s*([^\s]+)').Groups[1].Value)</CocoaVersion>
-    </PropertyGroup>
+  <Target Name="_SetupCocoaSDK">
```

Comment:
Condition removed?
> Condition="$([MSBuild]::IsOSPlatform('OSX')) And !Exists('$(SentryCocoaCache)Sentry-Dynamic.xcframework')"

Reply:
The outer targets had the condition but I've restored it for the inner target too 👍